src/App.tsx

```diff
@@ -38,8 +38,8 @@ function App() {
             <Route path="/signup">
               <Signup />
             </Route>
-            <Route path="/about">
-              <About />
+            <Route path="/home">
```

Comment:
use `/` instead of `/home`. `/home` is kinda ugly and the root document is implied home.